lib/actions/BackendAction.js

```diff
@@ -90,6 +93,16 @@ class BackendAction {
     })
   }
 
+  async writeExtensionConfigs () {
+    const attachedExtensions = this.appSettings.loadAttachedExtensions()
+    const promises = []
+    for (let extensionId in attachedExtensions) {
+      const config = attachedExtensions[extensionId]
```

Comment:
The content of the `config` is not the value of the `extension-config`
You need to read it separately.